tensorflow/lite/examples/label_image/BUILD

```diff
@@ -39,11 +39,11 @@ cc_binary(
         "@com_google_absl//absl/memory",
     ] + select({
         "//tensorflow:android": [
-            "//tensorflow/lite/delegates/gpu:gl_delegate",
+            "//tensorflow/lite/delegates/gpu:delegate",
```

Comment:
can you actually also add #include ".../delegate.h"?  it's being currently pulled in indirectly, but we should have explicit #include.

Reply:
OK. Thanks